src/main/java/com/epam/lab/paymentsystem/controller/CardController.java

```diff
@@ -48,6 +48,7 @@ public String getAccountPage(
       Model model) {
     LOGGER.info("Access to account page");
     List<Card> cards = cardService.getAllCardsByAccountId(id);
+    model.addAttribute("userLogin", login);
```

Comment:
What is it used for?

Reply:
It was deleted by mistake
It used by account html page for comparing current user with user on page